include/alpaka/math/MathUniformCudaHipBuiltIn.hpp

```diff
@@ -18,6 +18,7 @@
 #include <alpaka/math/Complex.hpp>
 #include <alpaka/math/Traits.hpp>
 
+#include <algorithm>
```

Comment:
Why is this required?

Reply:
Oh idk. @SimeonEhrig do you?

Reply:
That is a mistake. I forgot to remove it.